Code/Framework/AzCore/AzCore/Memory/IAllocator.h

```diff
@@ -62,6 +62,8 @@ namespace AZ
         virtual size_type               Capacity() const = 0;
         /// Returns max allocation size if possible. If not returned value is 0
         virtual size_type               GetMaxAllocationSize() const { return 0; }
+        /// Returns the maximum contiguous allocation size of a single allocation
+        virtual size_type               GetMaxContiguousAllocationSize() const = 0 { return 0; }
```

Comment:
Declaring function as virtual pure with implementation, please remove `= 0`.

Reply:
Ah this is code that I was using to force compilation errors for any classes that inherited from `IAllocatorAllocate` in order to determine which classes still needed to implement `GetMaxContiguousAllocationSize`

Will remove